### ISSUE TITLE
squelcher can't handle lists in extra

### ISSUE THREAD

**Issue by ghost** [certat/intelmq#163]
For events with fields like `extra.foo = [bar]`, the squelcher bot fails:

```python
Traceback (most recent call last):
  File "/home/aaron/intelmq/intelmq/lib/bot.py", line 176, in start
    self.process()
  File "/home/aaron/intelmq/intelmq/bots/experts/squelcher/expert.py", line 98, in process
    if set(condition.items()).issubset(event.items()) and all(conditions):
TypeError: unhashable type: 'list'
```

**Comment by ghost** [comment]
We need to convert the lists to tuples before calling `issubset`

**Comment by ghost** [comment]
Was fixed by fd927c612b54901dfb78a6a809bc5844db470bb3 Fri Jun 12
